commands/project/watch.ts

```diff
@@ -88,11 +88,11 @@ const handleUserInput = (accountId, projectName, currentBuildId) => {
 exports.handler = async options => {
   await loadAndValidateOptions(options);
 
-  const { initialUpload, path: projectPath, derivedAccountId } = options;
+  const { initialUpload, derivedAccountId } = options;
 
   trackCommandUsage('project-watch', null, derivedAccountId);
 
-  const { projectConfig, projectDir } = await getProjectConfig(projectPath);
+  const { projectConfig, projectDir } = await getProjectConfig();
```

Comment:
See my comment on the `upload` command.

Reply:
Added better copy in [6d99f67](https://github.com/HubSpot/hubspot-cli/pull/1266/commits/6d99f67a0643d7d87a8c75fc8f866891f70fcfeb)